testsuite/forge/src/backend/k8s/mod.rs

```diff
@@ -84,10 +84,14 @@ impl Drop for K8sFactory {
     // When the K8sSwarm struct goes out of scope we need to wipe the chain state and scale down
     fn drop(&mut self) {
         uninstall_from_k8s_cluster().unwrap();
-        set_eks_nodegroup_size(self.cluster_name.clone(), 0, true).unwrap();
+        let runtime = Runtime::new().unwrap();
```

Comment:
This could present some problems because this would require that the type isn't dropped inside of an async context.

Reply:
Any suggestions?

Reply:
No nothing atm, just wanted to call it out as something we may need to think about for forge specifically (although this was going to be a problem either way)